acceptance/analyzer_test.go

```diff
@@ -145,34 +127,57 @@ func setupAnalyzeFixtures(t *testing.T) analyzeFixtures {
 
 	// Registry
 
+	// setup read and write images
 	someAppName := "some-app-image-" + h.RandString(10)
 	fixtures.authRegAppImage, fixtures.authRegAuthConfig = buildAuthRegistryImage(
 		t,
 		someAppName,
 		filepath.Join("testdata", "analyzer", "app-image"),
+		testRegistry,
 		"--build-arg", "fromImage="+containerBaseImage,
 		"--build-arg", "metadata="+appMeta,
 	)
-	fixtures.readOnlyRegAppImage = readOnlyRegistry.RepoName(someAppName)
```

Comment:
Don't we still need this?

Reply:
Oh I see, now it's being set here: https://github.com/buildpacks/lifecycle/pull/685/files#diff-c4e4b4a55ebbd71443c5260020b287f3018cb7bb4c65c79e74554b7f97c45656R155

Reply:
IIUC, since we only have one registry now, we need to actually create the read only image instead of relying on access through the other registry to give the correct access permissions.